2024/json-ld-wg/index.html

```diff
@@ -166,11 +166,82 @@ <h2>Scope</h2>
         </p>
         <p>The Working group is expected to coordinate with the <a href='https://www.w3.org/community/json-ld/'>JSON for Linking Data Community Group</a> on consensus-based proposals related to content changes for the JSON-LD Working Group Deliverables. The Chairs of this group may choose to reject proposals that are incompatible with this Charter.</p>
 
+        <section id="section-feature-updates">
+          <h3 id="in-scope">In Scope</h3>
+            <p>In addition to Errata and sychronization with RDF-star, the following features tracked on the
```

Comment:
Is it correct that now RDF-star is going to be known as RDF 1.2 because the reification functionality is going to be embedded into RDF per se?

```suggestion
            <p>In addition to Errata and sychronization with RDF 1.2, the following features tracked on the
```

Reply:
The WG name is RDF-star, although they've indicated they will likely change to RDF 1.2 in the future. The specs will use the "1.2" branding. But, for now, it is the RDF-star WG we're synchronizing with.

Reply:
```suggestion
            <p>In addition to Errata and synchronization with the RDF-star Working Group, the following features tracked on the
```